pan-domain-auth-core/src/main/scala/com/gu/pandomainauth/model/PanDomainAuthSettings.scala

```diff
@@ -1,10 +1,11 @@
 package com.gu.pandomainauth.model
 
-import com.gu.pandomainauth.{PrivateKey, PublicKey}
+import com.gu.pandomainauth.service.Crypto
+
+import java.security.KeyPair
 
 case class PanDomainAuthSettings(
-  publicKey: PublicKey,
-  privateKey: PrivateKey,
+  signingKeyPair: KeyPair,
```

Comment:
The signature of the `PanDomainAuthSettings` case class has changed here, but thankfully this won't affect consumers of this library, as they all use the companion `apply()` method that accepts `settingMap: Map[String, String]`.